platform/ios/src/MGLMapView.mm

```diff
@@ -3326,12 +3326,10 @@ - (void)_setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count
     {
         latLngs.push_back({coordinates[i].latitude, coordinates[i].longitude});
     }
+    
+    CLLocationDirection cameraDirection = direction >= 0 ? direction : 0;
```

Comment:
This change is inconsistent with the meaning of negative direction elsewhere in the SDK and should be reverted: #14574.